docs/api.md

```diff
@@ -286,6 +287,12 @@ Emitted when Browser context gets closed. This might happen because of one of th
 - Browser application is closed or crashed.
 - The [`browser.close`](#browserclose) method was called.
 
+#### event: 'pageevent'
+- <[PageEvent]>
+
+Emitted when a new Page is created in the BrowserContext. The event will also fire for popup
+pages.
```

Comment:
This event will fire for popup pages, shift-clicks, etc.

Reply:
Will this event fire if you open a new tab with the browser ui?